sql/core/src/test/scala/org/apache/spark/sql/parquet/ParquetSchemaSuite.scala

```diff
@@ -478,11 +478,12 @@ class ParquetSchemaSuite extends ParquetSchemaTest {
     StructType(Seq(
       StructField(
         "f1",
-        ArrayType(IntegerType, containsNull = true),
+        ArrayType(
+          StructType(Seq(StructField("num", IntegerType))), containsNull = false),
```

Comment:
Hm, this change is supposed to be wrong. The Parquet schema below just represents an integer array. Don't know why it passed tests... Trying to figure this out.